skywalking-nginx-lua-1.0-0.rockspec

```diff
@@ -0,0 +1,29 @@
+package = "skywalking-nginx-lua"
+version = "1.0-0"
+source = {
+   url = "git://github.com/apache/skywalking-nginx-lua",
+   branch = "master",
```

Comment:
We'd better use a separate version branch, or git tag.

Reply:
Typically, SkyWalking will use tag to do release.